organizations/models.py

```diff
@@ -18,6 +20,15 @@ class Organization(TenantMixin, SafeDeleteModel):
     class Meta:
         db_table = "organization"
 
+    def _generate_random_cryptographically_secure_string(self, length=20):
```

Comment:
crypto word can come in the description.
```suggestion
    def _generate_random_secure_string(self, length=20):
```